staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -774,6 +773,7 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
 	handler = genericapifilters.WithCacheControl(handler)
 	handler = genericfilters.WithHSTS(handler, c.HSTSDirectives)
 	handler = genericfilters.WithHTTPLogging(handler)
+	handler = genericapifilters.WithRequestInfo(handler, c.RequestInfoResolver)
```

Comment:
For the curious, this reordering doesn't cause apiserver to do more net work unless the probabilistic GOAWAY triggers, which is going to be rare or extremely rare.

Reply:
SGTM